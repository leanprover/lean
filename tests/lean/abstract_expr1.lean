#abstract_expr 0 @zero
#abstract_expr 0 ℕ
#abstract_expr 0 nat_has_zero
#abstract_expr 0 @zero ℕ
#abstract_expr 0 @zero ℕ nat_has_zero
