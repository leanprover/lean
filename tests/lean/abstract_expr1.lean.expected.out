1558663523
3156312089
1320171407
1921459337
1547633539
